src/mono/wasm/build/WasmApp.targets

```diff
@@ -0,0 +1,65 @@
+<Project>
+  <UsingTask TaskName="WasmAppBuilder" AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
+  <UsingTask TaskName="WasmLoadAssembliesAndReferences" AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
+
+  <Target Name="WasmBuildApp" AfterTargets="Publish">
+    <!-- FIXME: TArgetOS check -->
```

Comment:
Can the check be added in this PR?

Reply:
Oops, forgot to remove this. I'm leaving the check up to the project importing this file.

Reply:
I added an `Error` to catch a case where this file was included incorrectly. Thanks.